curriculum/challenges/english/25-front-end-development/workshop-reusable-mega-navbar/6745d799beeb822076a4da9d.md

```diff
@@ -0,0 +1,174 @@
+---
+id: 6745d799beeb822076a4da9d
+title: Step 7
+challengeType: 0
+dashedName: step-7
+---
+
+# --description--
+
+Inside the third list item, create an anchor tag with its `href` attribute set to `#` and a `haspopup` attribute set to `true`. Also, set the text of the `a` tag to `Apps`.
+
+This will communicate to screen readers that the third list item is a popup.
```

Comment:
```suggestion
Inside the third list item, create a button and add the `aria-expanded` attribute set to `false`. Also, set the text of the `button` to `Apps`.

The `aria-expanded` attribute will communicate to screen reader users that the button triggers an expandable submenu.
```

It's `aria-haspopup`, but you would only use that with a true ARIA menu, which this is not. This is a simple navigation menu that does not need the enhanced ARIA attributes. A simple `aria-expanded` is all that is needed here to tell screen reader users that the item has a submenu. Also, you would use a button as the toggle element, not a link. The general rule is that links take you places and buttons do things. In this case, the purpose of this element is to hide/show the submenu. So it is doing something.

I'll let someone else make the necessary updates to the tests and other steps.